packages/utils/src/instrument/fetch.ts

```diff
@@ -65,6 +66,15 @@ function instrumentFetch(): void {
           };
 
           triggerHandlers('fetch', erroredHandlerData);
+
+          if (isError(error) && error.stack === undefined) {
+            // NOTE: If you are a Sentry user, and you are seeing this stack frame,
+            //       it means the error, that was caused by your fetch call did not
+            //       have a stack trace, so the SDK backfilled the stack trace so
+            //       you can see which fetch call failed.
+            error.stack = new Error(error.message).stack;
```

Comment:
do we need to trim the frames here? I think we can set `framesToPop` to do this (`error.framesToPop = 1`).